<launch>
    <node name="a1_mcp_server" pkg="a1_mcp_bridge" type="mcp_server.py" output="screen">
        <param name="server_port" value="8080" />
    </node>

    <param name="robot_description" command="$(find xacro)/xacro $(find a1_description)/xacro/robot.xacro"/>
</launch>
